src/vos/vos_ts.h

```diff
@@ -649,14 +649,13 @@ void
 vos_ts_set_upgrade(struct vos_ts_set *ts_set);
 
 /** Free an allocated timestamp set
+ *
+ * Implemented as a macro to improve logging.
  *
  * \param[in]	ts_set	Set to free
  */
-static inline void
-vos_ts_set_free(struct vos_ts_set *ts_set)
-{
-	D_FREE(ts_set);
-}
+
+#define vos_ts_set_free(ts_set) D_FREE(ts_set)
```

Comment:
This isn't 100% identical as it'll also set the pointer to NULL but that's probably an improvement anyway.